packages/salesforcedx-vscode-apex/src/commands/forceApexTestSuite.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { TestService } from '@salesforce/apex-node';
+import {
+  LibraryCommandletExecutor,
+  SfdxCommandlet,
+  SfdxWorkspaceChecker
+} from '@salesforce/salesforcedx-utils-vscode/out/src';
+import {
+  CancelResponse,
+  ContinueResponse,
+  ParametersGatherer
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import { readFileSync } from 'fs';
+import { basename } from 'path';
+import * as vscode from 'vscode';
+import { OUTPUT_CHANNEL } from '../channels';
+import { workspaceContext } from '../context';
+import { nls } from '../messages';
+import {
+  ApexLibraryTestRunExecutor,
+  ApexTestQuickPickItem,
+  TestType
+} from './forceApexTestRun';
+
+export type ApexTestSuiteOptions = { suitename: string; tests: string[] };
+
+async function listApexClassItems(): Promise<ApexTestQuickPickItem[]> {
+  const apexClasses = await vscode.workspace.findFiles('**/*.cls');
+  const apexClassItems: ApexTestQuickPickItem[] = [];
+
+  apexClasses.forEach(apexClass => {
+    const fileContent = readFileSync(apexClass.fsPath).toString();
+    if (fileContent && fileContent.toLowerCase().includes('@istest')) {
+      apexClassItems.push({
+        label: basename(apexClass.toString()).replace('.cls', ''),
+        description: apexClass.fsPath,
+        type: TestType.Class
+      });
+    }
+  });
+
+  return apexClassItems;
+}
+
+async function listApexTestSuiteItems(): Promise<ApexTestQuickPickItem[]> {
+  const connection = await workspaceContext.getConnection();
+  const testService = new TestService(connection);
+  const testSuites = await testService.retrieveAllSuites();
+
+  const quickPickItems = testSuites.map(testSuite => {
+    return {
+      label: testSuite.TestSuiteName,
+      // @ts-ignore
+      description: testSuite.Id,
```

Comment:
The field is named incorrectly in the apex-node package so we'll need to update that so we can remove the `ts-ignore` here